code/modules/antagonists/eldritch_cult/eldritch_effects.dm

```diff
@@ -0,0 +1,149 @@
+/obj/effect/eldritch
+	name = "Generic rune"
+	desc = "Weird combination of shapes and symbols etched into the floor itself. The indentation is filled with thick black tar-like fluid."
+	anchored = TRUE
+	icon_state = ""
+	resistance_flags = FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	layer = SIGIL_LAYER
+
+/obj/effect/eldritch/a
```

Comment:
👀

Reply:
i will fix all of these tommorow. im too tired rn